.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: 🤔 Questions and Help
+    url: https://discord.com/channels/747741246667227157/883355707146141737
```

Comment:
I created this permanent invite link for support/general channel: https://discord.gg/dT3hDTtABD

Reply:
Perfect! Thanks.